webapp/components/Registration/RegistrationSlider.vue

```diff
@@ -0,0 +1,235 @@
+    RegistrationItemEnterEmail,
+    RegistrationItemEnterInvite,
+    RegistrationItemEnterNonce,
+  },
+  props: {
+    registrationType: { type: String, required: true },
+    overwriteSliderData: { type: Object, default: () => {} },
+  },
+  data() {
+    const slidersPortfolio = [
+      {
+        name: 'enter-invite',
+        // title: this.$t('components.registration.create-user-account.title'),
+        title: 'Invitation', // Wolle
+        validated: false,
+        data: { request: null, response: { isValidInviteCode: false } },
+        button: {
+          title: 'Next', // Wolle
+          icon: 'arrow-right',
+          callback: this.buttonCallback,
+          sliderCallback: null, // optional set by slot
+        },
+      },
+      {
+        name: 'enter-email',
+        title: 'E-Mail', // Wolle
+        validated: false,
+        data: { request: null, response: null },
+        button: {
+          title: '', // set by slider component
+          icon: '', // set by slider component
+          callback: this.buttonCallback,
+          sliderCallback: null, // optional set by slot
+        },
+      },
+      {
+        name: 'enter-nonce',
+        title: 'E-Mail Confirmation', // Wolle
+        validated: false,
+        data: { request: null, response: null },
+        button: {
+          title: 'Confirm', // Wolle
+          icon: 'arrow-right',
+          callback: this.buttonCallback,
+          sliderCallback: null, // optional set by slot
+        },
+      },
+      {
+        name: 'create-user-account',
+        title: this.$t('components.registration.create-user-account.title'),
+        validated: false,
+        data: { request: null, response: null },
+        button: {
+          // title: this.$t('actions.save'), // Wolle
+          title: 'Create', // Wolle
+          icon: 'check',
+          callback: this.buttonCallback,
+          sliderCallback: null, // optional set by slot
+        },
+      },
+    ]
+    let sliders = []
+    switch (this.registrationType) {
+      case 'invite-code':
+        sliders = [
+          slidersPortfolio[0],
+          slidersPortfolio[1],
+          slidersPortfolio[2],
+          slidersPortfolio[3],
+        ]
+        break
+      case 'public-registration':
+        sliders = [slidersPortfolio[1], slidersPortfolio[2], slidersPortfolio[3]]
+        break
+      case 'invite-mail':
+        sliders = [slidersPortfolio[2], slidersPortfolio[3]]
+        break
+    }
+
+    return {
+      links,
+      metadata,
+      sliderData: {
+        collectedInputData: {
+          inviteCode: null,
+          email: null,
+          emailSend: null,
+          nonce: null,
+          name: null,
+          password: null,
+          passwordConfirmation: null,
+          about: null,
+          termsAndConditionsConfirmed: null,
+          dataPrivacy: null,
+          minimumAge: null,
+          noCommercial: null,
+          noPolitical: null,
```

Comment:
maybe we can put that into each slider and just have a array of sliderdata objects. This is not needed for this PR tho - just as reminder for refactoring

Reply:
I think it's not needed for code functionality anyways. For me it has the functionality to give an overview about all the data to collect at the top-level.